v2/permissions/checker.go

```diff
@@ -150,9 +150,8 @@ func aConditionIsMet(conditions []Condition, attributes map[string]string) bool
 }
 
 func conditionIsMet(condition Condition, attributes map[string]string) bool {
-	for _, attribute := range condition.Attributes {
-		value, ok := attributes[attribute]
-		if !ok {
+	for attribute, value := range attributes {
```

Comment:
Do we need to loop over the map? The original wasn't doing this. Would it not be sufficient just to remove the for loop and just do the original `value, ok := attributes[condition.Attribute]` and simply `return false` if `!ok`

Reply:
Ive reversed the logic in regards to looping through attributes instead of condition.attributes (as it was a slice). Secondly, I require the value for each iteration that is input to the second part of this method see lines 158 to 165. Otherwise the unit tests don't pass. Will have another look.